scapy/contrib/mqtt.py

```diff
@@ -44,9 +46,16 @@ def getfield(self, pkt, s):
                 raise Scapy_Exception("%s: malformed length field" %
                                       self.__class__.__name__)
 
+    def randval(self):
+        return RandVariableFieldLen()
 
-# LAYERS
 
+class RandVariableFieldLen(RandNum):
```

Comment:
You need to import `RandNum` (from `scapy.volatile`).

Reply:
Indeed. Fixed it